modules/attributes/pages/positional-and-named-attributes.adoc

```diff
@@ -56,3 +58,22 @@ No special processing is performed, except for attribute reference expansion, if
 
 If the attribute value contains the same quote character being used to enclose the value, escape the quote character in the value by prefixing it with a backslash (e.g., `title="\"Dark Horse\" is a song by George Harrison"`).
 // end::subs[]
+
+== Attribute parsing
+
+The exact rules for determining the boundaries of attributes and whether an attribute is positional or named are somewhat complicated.
```

Comment:
It's not helpful for the reader to start out by saying these rules are complicated. (Plus, it leaves room for AsciiDoc to be ridiculed). We should just say that a positional attribute must sometimes be enclosed in quotes so as not be misinterpreted as a named attribute.